src/components/Navbar/Sidebar.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+import { NavLink } from 'react-router';
+import Box from '@mui/material/Box';
+import Drawer from '@mui/material/Drawer';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import { IconButton, Stack, Typography } from '@mui/material';
```

Comment:
Keep imports consistent. If all other MUI imports are default imports, then you're not missing named exports.